ste/xfer-anyToRemote-file.go

```diff
@@ -361,15 +362,22 @@ func epilogueWithCleanupSendToRemote(jptm IJobPartTransferMgr, s sender, sip ISo
 
 	if jptm.IsLive() && info.DestLengthValidation {
 		_, isS2SCopier := s.(s2sCopier)
+		shouldCheckLength := true
 		destLength, err := s.GetDestinationLength()
 
-		if err != nil {
-			wrapped := fmt.Errorf("could not read destination length. If destination is write-only, use --check-length=false on the AzCopy command line. %w", err)
-			jptm.FailActiveSend(common.IffString(isS2SCopier, "S2S ", "Upload ")+"Length check: Get destination length", wrapped)
+		if resp, respOk := err.(pipeline.Response); respOk && resp.Response() != nil &&
+			resp.Response().StatusCode != http.StatusForbidden {
+			shouldCheckLength = false
 		}
+		if shouldCheckLength {
+			if err != nil {
```

Comment:
To make things clearer, I would suggest to move the error handling code together, and

1. If status is permission forbidden, we should write a message to the screen (using glcm.Info) just once, telling the user that we didn't have permission to check length, and the read permission is required.
2. If the error is anything else than permission problem, still write a message to the screen, but just say the length couldn't be obtained because of xxxx reason (no need to explain how to fix)

Reply:
Made these changes. Also, the permission-error will be displayed exactly once.